src/hdmint/wallet.cpp

```diff
@@ -442,7 +449,7 @@ int32_t CHDMintWallet::GetCount()
 void CHDMintWallet::ResetCount()
 {
     CWalletDB walletdb(strWalletFile);
-    walletdb.ReadZerocoinCount(nCountNextUse);
+    walletdb.WriteMintCount(nCountNextUse);
```

Comment:
The old one is read not write. Is this intentional?

Reply:
Yes this is intentional, it was a bug.